pandas/tests/groupby/test_groupby.py

```diff
@@ -1385,7 +1408,7 @@ def foo(x):
 
 
 def test_group_name_available_in_inference_pass():
-    # gh-15062
+    # gh-15062, issue #21609
```

Comment:
If all double-runs (and not just the ones with output `index` attribute) were fixed, this test wouldn't have passed. So I included the issue number.